src/lib/viewers/controls/color-picker/ColorPickerControl.scss

```diff
@@ -9,6 +9,15 @@
 
 .bp-ColorPickerControl-button {
     @include bp-ControlButton;
+
+    &:hover,
+    &:focus {
+        .bp-ColorPickerControl-highlight {
+            padding: 8px;
+            background-color: $fours;
+            border-radius: 4px;
+        }
+    }
```

Comment:
Is this enough to cover the case when we use keyboard tab to access the color palette and focus leaves the button?

Reply:
Good catch - I've added another change for reusing `isColorPickerToggled` to determine if we should also be showing the sub shadow when the palette is open.